glib/src/value.rs

```diff
@@ -565,6 +565,22 @@ impl Value {
         }
     }
 
+    // rustdoc-stripper-ignore-next
+    /// Tries to transform the value into the target type
+    #[doc(alias = "g_value_transform")]
+    pub fn get_transformed<T: ValueType>(&self) -> Result<T, crate::BoolError> {
```

Comment:
`to_transformed()` maybe? It's a conversion, not a getter.

@pbor what do you think?

Reply:
Is it really more convenient than `v.transform().get()`?

Reply:
Maybe if we want a shortcut we should implement `TryInto` ?

Reply:
Another naming possibility is `transform_to` which has a precedent: `https://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib/struct.BindingGroupBuilder.html#method.transform_to` though that actually returns `Option<Value>`

Reply:
the other option is to rename this function to `transform` and rename the other `transform` function to `transform_as_value` or something like that